### PR TITLE
[KED- 2522] Removed build/api after running build

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

This PR is created to resolve the issue highlighted in detail [here](https://jira.quantumblack.com/browse/KED-2522?jql=assignee%20%3D%20currentUser()%20AND%20resolution%20%3D%20Unresolved%20AND%20project%20%3D%20KED%20ORDER%20BY%20updated%20DESC)

## Development notes

Added remove file command to package.json and makefile 

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "npm run build:css && react-scripts build",
     "build:css": "node-sass src/ -o src/",
+    "postbuild":"rm -rf build/api",
```
Comment (limdauto) [inline package.json:13]: If we add this `postbuild` script, do we need to run `rm` manually?

Comment (rashidakanchwala) [reply package.json:13]: nope, it runs automatically when you npm run build, i tried it

Comment (limdauto) [reply package.json:13]: then can we not include the change in the `Makefile`? :D

Comment (rashidakanchwala) [reply package.json:13]: uhh i read this wrong... i thot u meant can't we do the above in makefile... hence the discussion. Sorry!

**Review by limdauto — APPROVED** [review]

**Pull request merged**